app/models/anime.rb

```diff
@@ -260,105 +335,103 @@ def self.create_or_update_from_hash(hash)
       character = Character.create_or_update_from_hash ch
       if ch[:voice_actors].length > 0
         ch[:voice_actors].each do |actor|
-          Casting.create_or_update_from_hash actor.merge({
+          Casting.create_or_update_from_hash actor.merge(
             featured: ch[:featured],
             character: character,
             castable: anime
-          })
+          )
         end
       else
-        Casting.create_or_update_from_hash({
+        Casting.create_or_update_from_hash(
           featured: ch[:featured],
           character: character,
           castable: anime
-        })
+        )
       end
     end
     anime
   end
 
   def show_type_enum
-    ["OVA", "ONA", "Movie", "TV", "Special", "Music"]
+    self::SHOW_TYPES
   end
 
   def status
-    # If the started_airing_date is in the future or not specified, the show hasn't
-    # aired yet.
-    if started_airing_date.nil? or started_airing_date > Time.now.to_date
-      return "Not Yet Aired"
+    # If the started_airing_date is in the future or not specified,
+    # the show hasn't aired yet.
+    if started_airing_date.nil? || started_airing_date > Time.zone.now.to_date
+      return 'Not Yet Aired'
     end
 
-    # Since the show's airing date is in the past, it has either "Finished Airing" or
-    # is "Currently Airing".
-
     # If the show has only one episode, then it has "Finished Airing".
-    if episode_count == 1
-      return "Finished Airing"
-    end
+    return 'Finished Airing' if episode_count == 1
 
-    # If the finished_airing_date is specified and in the future, the the show is
-    # "Currently Airing". If it is specified and in the past, then the show has
-    # "Finished Airing". If it is not specified, the show is "Currently Airing".
+    # If the finished_airing_date is specified and in the future,
+    # the the show is "Currently Airing".
+    # If it is specified and in the past, then the show has "Finished Airing".
+    # If it is not specified, the show is "Currently Airing".
     if finished_airing_date.nil?
-      return "Currently Airing"
+      return 'Currently Airing'
     else
-      if finished_airing_date > Time.now.to_date
-        return "Currently Airing"
+      if finished_airing_date > Time.zone.now.to_date
+        return 'Currently Airing'
       else
-        return "Finished Airing"
+        return 'Finished Airing'
       end
     end
   end
 
   before_save do
     # If episode_count has increased, create new episodes.
-    if self.episode_count and self.episodes.length < self.episode_count and (self.episodes.length == 0 or self.thetvdb_series_id.nil?)
-      (self.episodes.length+1).upto(self.episode_count) do |n|
-        Episode.create(anime_id: self.id, number: n)
+    if episode_count &&
```

Comment:
Hmm.  So we can't just wrap it at 80 at the second `&&`?  :/ I'll have to check if there's a flag in rubocop for that